modules/service-ca-certificates.adoc

```diff
@@ -0,0 +1,80 @@
+// Module included in the following assemblies:
+//
+// * authentication/certificates/certificate-reference.adoc
+
+[id="service-ca-certificates_{context}"]
+= Service CA certificates
+
+[discrete]
+== Purpose
+
+`service-ca` is an Operator that creates a self-signed CA, which is valid for
+one year, when an {product-title} cluster is deployed.
```

Comment:
Need to confirm the validity length

Reply:
Any release that includes automated rotation (4.4 and greater, some 4.2.z and 4.3.z releases) will have a validity duration of 26 months.